go.mod

```diff
@@ -105,7 +105,7 @@ require (
 	github.com/hashicorp/vault-plugin-secrets-openldap v0.4.0
 	github.com/hashicorp/vault-plugin-secrets-terraform v0.1.0
 	github.com/hashicorp/vault/api v1.0.5-0.20210210214158-405eced08457
-	github.com/hashicorp/vault/sdk v0.1.14-0.20210505171055-299f311fa707
+	github.com/hashicorp/vault/sdk v0.2.0
```

Comment:
@calvn is this correct? Isn't the commit this is replacing later than 0.2.0 of the SDK?

Reply:
Yeah, can this be reverted?

Reply:
Actually, this may not matter because of the replace directive at the very top.